README.md

```diff
@@ -40,18 +40,22 @@ libraryDependencies += compilerPlugin(repo %% "akka-serializability-checker-plug
 
 ### 2. Dump Event Schema
 
-An sbt plugin that allows for dumping schema of events to a file. Can be used for detecting accidental changes of
+A sbt plugin that allows for dumping schema of events to a file. Can be used for detecting accidental changes of
 events.
 
 Installation:
 
 Add to `plugins.sbt`:
 ```scala
-libraryDependencies += repo % "sbt-dumpschema" % commit
+libraryDependencies += repo % "sbt-dump-event-schema" % commit
```

Comment:
Actually, I'd repeat the jitpack'd organization name explictly everywhere instead of using `repo` (also, this should've been `org`, not `repo`) for clarity

Reply:
Done it, not sure if it improved clarity. Because of increased length some code snippets have acquired navigation bar.

Reply:
@PawelLipski

Reply:
Okay, I still like the version with verbatim org name more, at least users don't need to look around the README for the missing pieces as much